listenbrainz/labs_api/labs/api/recording_from_recording_mbid.py

```diff
@@ -45,6 +46,7 @@ def fetch(self, params, offset=-1, count=-1):
                 args = [tuple([psycopg2.extensions.adapt(p) for p in mbids])]
                 curs.execute(query, tuple(args))
 
+                # Build an index with all redirected recordings
                 redirect_index = {}
                 while True:
                     row = curs.fetchone()
```

Comment:
FYI: psycopg2 cursors are iterable, so you don't have to do while True/fetch row/check none. You can just do `for row in curs`

Reply:
Ah, yes, thanks for the reminder -- I remember adopting this pattern eons ago when I encountered a bug in psycopg2 when I was a small child...